hrabsence/CRM/HRAbsence/BAO/HRAbsenceType.php

```diff
@@ -117,6 +77,59 @@ public static function create($params) {
   }
 
   /**
+   * Creates Activitu Types based on $params and $activityTypes
+   * @param  $activityTypes Array
```

Comment:
For PHP code we follow [Drupal's code standard](https://www.drupal.org/docs/develop/standards/coding-standards). These are the rules specific for docblocks: https://www.drupal.org/node/1354

- There should be a space between the method description and the @param section
- There should be a space between the @param and the @return section
- The correct order is @param TYPE $paramName Description
- Types are always all in lowercase (Except for objects). So it should be `array` instead of `Array` and `boolean` instead of `Boolean`